webapp/src/components/CreateEstatePage/EditEstate/EstateModal/EstateModal.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid, Header } from 'semantic-ui-react'
+import ParcelPreview from 'components/ParcelPreview'
+import { coordsType } from 'components/types'
+import { t } from 'modules/translation/utils'
+import './EstateModal.css'
+import { calculateZoomAndCenter } from 'modules/estates/utils'
```

Comment:
Nitpick mode on: Can we reorder this so `import './EstateModal.css'` is the last import? for consistency sake

Reply:
Agree on this, moved 3b80f5b